packages/calling-component-bindings/src/utils/callUtils.ts

```diff
@@ -11,8 +11,16 @@ import { memoizeFnAll, toFlatCommunicationIdentifier } from '@internal/acs-ui-co
  *
  * @internal
  */
-export const _isInCall = (callStatus?: CallStatus): boolean =>
-  !!callStatus && !['None', 'Disconnected', 'Connecting', 'LocalHold', 'RemoteHold', 'Ringing'].includes(callStatus);
+export const _isInCall = (callStatus?: CallStatus): boolean => {
+  /* @conditional-compile-remove(PSTN-calls) */ /* @conditional-compile-remove(one-to-n-calling) */
+  return (
+    !!callStatus && !['None', 'Disconnected', 'Connecting', 'LocalHold', 'Ringing', 'EarlyMedia'].includes(callStatus)
+  );
+  return (
+    !!callStatus &&
+    !['None', 'Disconnected', 'Connecting', 'LocalHold', 'RemoteHold', 'Ringing'].includes(callStatus as string)
```

Comment:
intentional that now `EarlyMedia` will mean isInCall is true? ( don't have a good grasp on what early media means)

Reply:
so the hold states and ringing shouldn't be part of the stable version of this function since they were originally not there. 

Isn't the logic of this function @JamesBurnside that if the state is in the array the user is not in the call? just the negation of that check?

Reply:
Oop thanks, I read the conditional compile the wrong way around!

Reply:
I would probably wouldn't even conditionally compile this and just update the isInCall - looks like its simply fixing a bug where we aren't checking the states correctly!

Reply:
Hi @anjulgarg and @dmceachernmsft , 
Can you please clarify on EarlyMedia details?

Reply:
EarlyMedia will only occur during PSTN calls.
It basically means that media is being received by the caller.
For example, you call a number and their phone starts ringing.
Media starts flowing from their device to yours telling you that their phone is ringing.
That's EarlyMedia. You can think of it as a 'Ringing' state.